db/seeds.rb

```diff
@@ -20,4 +25,4 @@
   end
 end
 
-AdminUser.create!(email: 'admin@example.com', password: 'password', password_confirmation: 'password')
+AdminUser.first_or_create!(email: 'admin@example.com', password: 'password', password_confirmation: 'password')
```

Comment:
Do we really need `first_or_create` in seeds.rb? It is only run in an empty database, right?